app/code/Magento/CatalogSearch/Model/Indexer/Fulltext/Action/DataProvider.php

```diff
@@ -205,7 +205,7 @@ public function getSearchableAttributes($backendType = null)
             $attributes = $productAttributes->getItems();
 
             $this->eventManager->dispatch(
-                'catelogsearch_searchable_attributes_load_after',
+                'catalogsearch_searchable_attributes_load_after',
```

Comment:
While this change is perfectly valid, 3-rd party extensions may already be relying on this misspelled event name causing their customizations to break.
For additional reference please check [Backward Compatible Development Guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html)

Reply:
Hi @ishakhsuvarov,

Thanks for your feedback. I missed that and you are right, it should be backwards compatible. I have changed this in the new commit.